manifests/t/Tencent/QQMusic/19.33/Tencent.QQMusic.installer.yaml

```diff
@@ -0,0 +1,62 @@
+# Created with YamlCreate.ps1 v2.2.5 $debug=QUSU.CRLF.7-3-4.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Tencent.QQMusic
+PackageVersion: "19.33"
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: silence=1
+  SilentWithProgress: silence=1
+  InstallLocation: installdir="<INSTALLPATH>"
+  Custom: runafterinstall=no
+UpgradeBehavior: install
+FileExtensions:
+- aac
+- ac3
+- aif
+- aiff
+- ape
+- au
+- caf
+- cue
+- dff
+- dsf
+- dts
+- flac
+- lrc
+- m4a
+- m4r
+- mflac
+- mgg
+- mid
+- mp3
+- mpc
+- ogg
+- opus
+- qmc0
+- qmc2
+- qmc3
+- qmc4
+- qmc6
+- qmc8
+- qmcflac
+- qrc
+- qsc3
+- tac
+- tkm
+- tta
+- wav
+- wma
+- wv
+ReleaseDate: 2023-05-17
+Installers:
+- Architecture: x86
+  InstallerUrl: https://dldir1v6.qq.com/music/clntupate/QQMusic_Setup_1933.exe
```

Comment:
```suggestion
  InstallerUrl: https://dldir1.qq.com/music/clntupate/QQMusic_Setup_1933.exe
```

Reply:
Do I still need to commit this suggestion as the URL has been waived?

Reply:
Apologies, yes please update the InstallerUrl. I added the waiver in err. I wasn't quite able to navigate from the PackageUrl to the InstallerUrl, - the URL is very similar, but the package name is slightly different. 

I'm unsure of the significance of those extra 2 characters (`v6`) in the domain, but it's probably better to use the same domain as the other download links. This domain might be for internal testing or another purpose, and thus might be unexpectedly depreciated or reused in a way that breaks this link. But the domain from their other download links has a higher probability of notifying the users of any changes, and of making fewer changes to avoid such complicated communication.

Reply:
I guess "v6" means IPv6? 🤔 Anyway, I got this URL from their updating API.
![image](https://github.com/microsoft/winget-pkgs/assets/56779163/935404bf-cc2b-492c-8501-2302fe67e452)